src/prog_algs/metrics/toe_profile_metrics.py

```diff
@@ -41,3 +43,42 @@ def alpha_lambda(toe_profile : ToEPredictionProfile, ground_truth : dict, lambda
                     print('\ttoe:', toe.key(key))
                     print('\tBounds: [{} - {}]({}%)'.format(lower_bound, upper_bound, toe.percentage_in_bounds([lower_bound, upper_bound])[key]))
             return result
+
+def prognostic_horizon(toe_profile : ToEPredictionProfile, criteria_eqn : Callable, ground_truth : dict, **kwargs):
+    """
+    Compute prognostic horizon metric, given by the difference between a time ti, when the predictions meet specified performance criteria, and the time corresponding to the end of life (EoL).
+    PH = EOL - ti
+    Args:
+        toe_profile (ToEPredictionProfile): A profile of predictions, the combination of multiple predictions
+        criteria_eqn (Callable function): A function calculating whether ground truth meets some criteria for corresponding key in ToEPredictionProfile
+        ground_truth (dict): Dictionary containing ground truth; specified as key, value pairs for event and its value
+        kwargs (optional): configuration arguments. Accepted args include:
+            * keys (list[string], optional): list of keys to use. If not provided, all keys are used.
```

Comment:
We don't use keys configuration argument. Instead, we just use print. Please replace this with a description of print